src/localization/captions/README.md

```diff
@@ -77,6 +63,11 @@ To start playing a caption, you call start. You can pass a start time in as an o
 ```javascript
   captionPlayer.start('welcome', 1200);
 ```
+
+
+```javascript
+  captionPlayer.start('welcome', 1200, {food: 'spring roll'});
```

Comment:
I'm guessing that the `welcome` key for captions might have a variable key called `food` in it. However, I think it would be helpful to have an example of what that captions text might look like.